editor/plugins/style_box_editor_plugin.cpp

```diff
@@ -58,7 +47,7 @@ void StyleBoxPreview::edit(const Ref<StyleBox> &p_stylebox) {
 	}
 	stylebox = p_stylebox;
 	if (p_stylebox.is_valid()) {
-		preview->add_theme_style_override("panel", stylebox);
+		add_theme_style_override("panel", stylebox);
 		stylebox->connect("changed", callable_mp(this, &StyleBoxPreview::_sb_changed));
```

Comment:
We could probably drop the `_sb_changed` methods and call `queue_redraw` directly wherever it was used.